### PR TITLE
Support for Passive Gait Recording

### PR DESCRIPTION
**Pull request by kalperin**
This pull request has support for displaying the preferences for passive data recording consent as well as updated null handling.  Gradle is used to build SageResearch-Android as a dependency via a local repository.  If SageResearch-Android is published to a maven repo then this grade file can be updated.

### CODE REVIEW

File: sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/profile/ProfileSettingsRecyclerViewAdapter.kt [hunk 1]
```diff
@@ -193,10 +203,32 @@ class ReportProfileItemRow(val reportProfileDataItem: ReportProfileDataItem,
                            profileItem: ProfileItemProfileTableItem,
                            profileDataLoader: ProfileDataLoader) : ProfileItemRow(profileItem, profileDataLoader) {
 
+    override val detail: String?
+        get() {
+            var value = profileDataLoader.getValueString(profileItem.profileItemKey)
```
Comment (kalperin) [inline sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/profile/ProfileSettingsRecyclerViewAdapter.kt:208]: Is there a `profileDataLoader.getValueBool()` that would standardize for different representations of a `bool`.  If not, then let's use `Boolean.parseBool()`

**Comment by kalperin** [comment]
Need to wait for the SageReseach-Android push before we approve, but this looks good.

File: build.gradle [hunk 2]
```diff
@@ -54,6 +54,12 @@ buildscript {
     classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
     classpath "com.jakewharton:butterknife-gradle-plugin:$butterknife_version"
 
+
+    classpath "org.sagebionetworks.research:domain:$sage_research_version"
+    classpath "org.sagebionetworks.research:presentation:$sage_research_version"
+    classpath "org.sagebionetworks.research:data:$sage_research_version"
```
Comment (nategbrown9) [inline build.gradle]: Shouldn't need to add these here. The sub modules that need sage research depend on them directly.

**Review by nategbrown9 — CHANGES_REQUESTED** [review]
Just need to remove the added dependencies in the top level gradle file.

File: build.gradle (hunk 2 above)
Comment (elkasvirElka) [reply build.gradle]: Fixed

**Review by nategbrown9 — APPROVED** [review]

**Pull request merged**